packages/app/src/server/models/attachment.js

```diff
@@ -35,6 +35,11 @@ module.exports = function(crowi) {
     createdAt: { type: Date, default: Date.now },
     temporaryUrlCached: { type: String },
     temporaryUrlExpiredAt: { type: Date },
+    attachmentType: {
+      type: String,
+      enum: ['BRAND_LOGO', 'WIKI_PAGE', 'USER_PAGE', 'PROFILE_IMAGE', 'UNCATEGORIZED'],
+      default: 'UNCATEGORIZED',
+    },
```

Comment:
このデータ、用意するならちゃんと活用しないと。今中途半端過ぎる。

- 用意するならまず union にする
    - USER_PAGE はなんだ？使わないタイプは消す
    - UNCATEGORIZED は undefined でいいと思う
- 今までのアップロード済みデータはどうするのか、migration しないのか
    - しないのだとすると'WIKI_PAGE' と 'PROFILE_IMAGE' は活用の術がない

Reply:
If we are gointo use the data, you need to do the following:
- First make it union.
  - Delete USER_PAGE as it will not be used
  - to make it union, define the type for the enum outside of the attachment.js (as it is not a ts file) then import it to use there
    - Refer to the `type ActionType` defined in `packages/app/src/server/interfaces/page-operation.ts`
  - `undefined` should be better instead of `UNCATEGORIZED `
- Need migration for all the data that has been uploaded so far. Otherwise the 'WIKI_PAGE' and 'PROFILE_IMAGE' are meaningless

Reply:
- Removed USER_PAGE from attachmentType
- Created attachmentType enum definition outside of attachment.js
- Replace UNCATEGORIZED with undefined